ttmar_starter.html

```diff
@@ -61,5 +61,66 @@
 
 </body>
 <script>
+var firstEvent = document.querySelector("#first_paragraph");
+var firstBlock = document.querySelector(".first_paragraph");
 
+firstEvent.addEventListener("click", function(){
+  console.log("hi");
+  firstBlock.style.background= "#000";
+})
+
+
+var secondEvent = document.querySelector("#second_paragraph");
+var secondBlock = document.querySelector(".second_paragraph");
+
+secondEvent.addEventListener("click", function(){
+  secondBlock.style.display="block";
+})
+
+
+var thirdEvent = document.querySelector("#first_chorus");
+var thirdBlock = document.querySelector(".first_chorus");
+
+thirdEvent.addEventListener("click", function(){
+  event.preventDefault();
+  thirdBlock.style.display="block";
+})
+
+
+var fourthEvent = document.querySelector("#third_paragraph");
+var fourthBlock = document.querySelector(".third_paragraph");
+
+fourthEvent.addEventListener("click", function(){
+  event.preventDefault();
+  for (i=0; i < 4; i++){
```

Comment:
Ohhhh, I see you used `querySelector` instead of `querySelctorAll`. The way you did it you really didn't have a choice, but I'd suggest looking at the solution branch's approach of using `querySelectorAll` here, and then iterating over the *array* of elements. A bit more flexible and maintainable.